### PR TITLE
RUMM-335 Create constants for Attributes

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Create public constants

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+package com.datadog.android.rum
+
+import com.datadog.android.Datadog
+import com.datadog.android.DatadogConfig
+
+/**
+ * This class holds constant rum attribute keys.
+ */
+// @Suppress("unused")
```
Comment (mariusc83) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt]: Leftover ?

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt [hunk 2]
```diff
@@ -19,14 +23,14 @@ import okhttp3.Response
  * (url, method, status code, optional error).
  *
  * If you use multiple Interceptors, make sure that this one is called first.
- * If you also use the [TracingInterceptor], make it is called after this one.
+ * If you also use the [TracingInterceptor], make sure it is called before this one.
```
Comment (mariusc83) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt]: Oh....this is something I am pretty sure they will miss. Maybe we can do a combined Interceptor later which uses the TracingInterceptor and RumInterceptor.

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt [hunk 3]
```diff
@@ -68,6 +78,24 @@ class RumInterceptor : Interceptor {
         }
     }
 
+    private fun extractTraceId(request: Request): String? {
```
Comment (mariusc83) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt]: cool

Comment (mariusc83) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt]: do you have a UnitTest for this case ?

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt (hunk 2 above)
Comment (xgouchet) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt]: My thoughts exactly, I added a task for that

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt (hunk 3 above)
Comment (xgouchet) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt]: Good call

File: dd-sdk-android/src/main/kotlin/com/datadog/android/log/LogAttributes.kt [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+    const val DD_TRACE_ID: String = "dd.trace_id"
+
+    /**
+     * A duration of any kind in nanoseconds. (Number)
+     * E.g.: HTTP response time, database query time, latency, etc.
+     */
+    const val DURATION: String = "duration"
+
+    /**
+     * The error type or kind (or code is some cases). (String)
```
Comment (ncreated) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/log/LogAttributes.kt]:
```suggestion
     * The error type or kind (or code in some cases). (String)
```

File: dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+    /**
+     * A duration of any kind in nanoseconds. (Number)
+     * This value is filled automatically by the [RumMonitor] for
+     * Views, Resources and Actions.
+     */
+    const val DURATION: String = "duration"
+
+    /**
+     * The error type or kind (or code is some cases). (String)
```
Comment (ncreated) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt]:
```suggestion
     * The error type or kind (or code in some cases). (String)
```

**Pull request merged**